Makefile

```diff
@@ -1,23 +1,58 @@
 CONTAINER_NAME := jenkins-server-container
 
 install:
-	./install.sh
+	./scripts/install.sh
 
 clean:
 	docker rm -v $$(docker ps -a -q | grep -v "$$(docker ps -q | xargs | sed 's/ /\\\|/g') ") 2>/dev/null || echo Nothing to do
 	docker rmi $$(docker images --no-trunc | grep none | awk '{print $$3 }') 2>/dev/null || echo Nothing to do
 
 build:
-	./build.sh
+	./scripts/build.sh
 
 start:
-	docker run --name $(CONTAINER_NAME) -p 8080:8080 -v /var/jenkins_home:/var/jenkins_home --restart always -d jenkins_server
+	docker run -d \
+		--name $(CONTAINER_NAME) \
+		-p 8080:8080 \
+		-v /var/jenkins_home:/var/jenkins_home \
+		-v /mnt/backup:/mnt/backup \
+		-v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
Hmmm... I think I got that... it responds my other question above.

It might be a problem on `boot2docker` kind of environments. Isn't it.